Rewrite Beatmaps article (#3115)
ppy/osu-wiki: wiki/Beatmaps/en.md

# Beatmaps

*Not to be confused with [Difficulties](/wiki/Difficulties).*

A beatmap is a set of difficulties in [osu!](/wiki/osu!_glossary). Beatmaps consist of a song, a background image and multiple [hit objects](/wiki/hit_objects) in sync with the beat. They can also include a video, and/or [storyboard](/wiki/storyboards).

> Beatmaps are shared through a system called [BSS](/wiki/BSS) (Beatmap Submission System). After beatmaps have been shared they will show up on the [Beatmaps Listing](https://osu.ppy.sh/beatmapsets) page on osu! web where they can be downloaded and favorited.

Reviewer: ```suggestion
Beatmaps are shared through a system called [BSS](/wiki/BSS) (Beatmap Submission System). After beatmaps have been shared, they will show up on the [Beatmaps Listing](https://osu.ppy.sh/beatmapsets) page on the website where they can be downloaded and favourited.
```

You can also find beatmaps in [beatmap packs](https://osu.ppy.sh/beatmaps/packs) or through osu!direct.<!--TODO: Add link once osu!supporter article has been written-->

## Beatmap Categories

> Beatmaps are split into 5 different categories depending on their recent activity, nomination status and/or favorites. The categories are:

Reviewer: ```suggestion
Beatmaps are split into 5 categories depending on their recent activity, nomination status, or favorites. The categories are:
```

- Graveyard

> - Work in Progress/Pending

Reviewer: ```suggestion
- Work in Progress and Pending
```
using the slash there makes it seem that both categories are aliases imo

- Qualified
- Ranked
- Loved

### Graveyard

> Graveyarded beatmaps are beatmaps that have been abandoned by their creator. Beatmaps are moved to the Graveyard from the [pending](#pending) category automatically once they haven't been updated for 4 weeks. Graveyarded beatmaps do not have leaderboards and will not affect user statistics, but can still be downloaded and played like normally.

Reviewer: ```suggestion
Graveyarded beatmaps are beatmaps that have been abandoned by their creator. Beatmaps are moved to the Graveyard category—from the [Pending](#pending) category—automatically once they haven't been updated for 4 weeks. Graveyarded beatmaps do not have leaderboards and will not affect user statistics. However, they can still be downloaded and played like normally.
```

> Once a beatmap is graveyarded it can be revived (put back into pending) by updating the beatmap through the [BSS](/wiki/BSS) (Beatmap Submission System)

Reviewer: ```suggestion
Once a beatmap is graveyarded, it can be revived (i.e. being it back to the pending category). This can be done by updating the beatmap via [BSS](/wiki/BSS) (Beatmap Submission System).
```

Graveyarded beatmaps use the question mark icon (![Question mark icon](/wiki/shared/status/graveyard.png)) in the song selection screen.

> ### Pending/Work in Progress

Reviewer: ```suggestion
### Work in Progress and Pending
```

Reviewer: If the suggestion on line 16 was declined, please use this instead:
```suggestion
### Work in Progress/Pending
```
just to keep it consistent with line 16

> Pending/Work in Progress (WIP) are the two standard categories beatmaps are put into on initial submission. Beatmaps in these categories can move to the qualified category upon nomination by at least two [Beatmap Nominators](/wiki/Beatmap_Nominators). This usually includes a great amount of feedback from [modders](/wiki/modders) through [modding](/wiki/modding). Pending/WIP beatmaps do not have leaderboards and will not affect user statistics.

Reviewer: ```suggestion
Work in Progress (commonly initialised as *WIP*) and Pending are the two standard categories beatmaps are put into on initial submission. Beatmaps in these categories can move to the [Qualified](#qualified) category upon nomination by at least two [Beatmap Nominators](/wiki/Beatmap_Nominators). This usually includes a great amount of feedback from [modders](/wiki/modders) through [modding](/wiki/modding). Work in Progress and Pending beatmaps do not have leaderboards and will not affect user statistics.
```

> Inactive beatmaps that have been in this category for 4 weeks are automatically moved into the [graveyard](#graveyard) status.

Reviewer: ```suggestion
Inactive beatmaps that have been in this category for 4 weeks are automatically moved into the [Graveyard](#graveyard) status.
```

> Pending/WIP beatmaps use the question mark icon (![Question mark icon](/wiki/shared/status/pending.png)) in the song selection screen.

Reviewer: ```suggestion
Work in Progress and Pending beatmaps use the question mark icon (![Question mark icon](/wiki/shared/status/pending.png)) in the song selection screen.
```

### Qualified

Qualified beatmaps are in the last stage of the [Beatmap ranking procedure](/wiki/Beatmap_ranking_procedure). These beatmaps have been nominated by at least two [Beatmap Nominators](/wiki/Beatmap_Nominators) who have made sure that the beatmap meets the standards of the [Ranking Criteria](/wiki/ranking_criteria).

> Beatmaps stay qualified for a period of approximately 7 days before they enter the [Ranked]((#ranked)) section permanently and become endorsed as official content. This period serves as a last line of defense in which any flaws and mistakes that might have slipped through can be fixed. In this period, beatmaps can be disqualified, changed and re-qualified without reseting the time it takes for the beatmap to become ranked. The Qualified section is in that sense, the last step before being considered a fully fledged beatmap.

Reviewer: ```suggestion
Beatmaps stay qualified for 7 days before they enter the [Ranked](#ranked) category permanently and become endorsed as official content. This period serves as a last line of defense in which any flaws and mistakes that might have slipped through can be fixed. In this period, beatmaps can be disqualified, changed, and re-qualified without resetting the time it takes for the beatmap to become ranked. The Qualified category is—in that sense—the last step before being considered a fully fledged beatmap.
```

Qualified beatmaps have leaderboards. Though, upon playing the beatmap a dialogue will display, noting that:

- No [performance points (pp)](/wiki/Performance_Points) will be awarded.
- All scores will be deleted when the status is changed.

Qualified beatmaps use the check icon (![Check icon](/wiki/shared/status/qualified.png)) in the song selection screen.

### Ranked

> Ranked beatmaps are endorsed as official content, in the sense that they have reached or surpassed the standards of the [Ranking Criteria](/wiki/ranking_criteria). This is to ensure that most players can expect a certain kind of standard while playing them. Ranked beatmaps allow other players to compete on leaderboards and to gain performance points from setting scores on them. Once a beatmap has achieved Ranked status, its status cannot be altered again (except for extraordinary cases).

Reviewer: ```suggestion
Ranked beatmaps are endorsed as official content, in the sense that they have reached or surpassed the standards of the [Ranking Criteria](/wiki/ranking_criteria). This is to ensure that most players can expect a certain kind of standard while playing them. Ranked beatmaps allow other players to compete on leaderboards and to gain performance points from setting scores on them. Once a beatmap has achieved ranked status, its status cannot be altered again (except for extraordinary cases).
```
I lower-cased the "r" in "Ranked status" because I think it's talking about a status and not the category.

Ranked beatmaps use the double upwards chevron icon (![Double upwards chevron icon](/wiki/shared/status/ranked.png)) in the song selection screen.

### Loved

*For history of the Loved category, see: [History of Loved](/wiki/Beatmaps/History_of_Loved).*

> Loved beatmaps may not meet the [ranking criteria](/wiki/Ranking_Criteria), but are *loved* by the community and thus are regularly added based on community votes in [Project Loved](https://osu.ppy.sh/community/forums/120). Before the beatmap begins playing, a dialogue will display, noting that:

Reviewer: ```suggestion
Loved beatmaps may not meet the [Ranking Criteria](/wiki/Ranking_Criteria), but are *loved* by the community and thus are regularly added based on community votes in [Project Loved](https://osu.ppy.sh/community/forums/120). Before the beatmap begins playing, a dialogue will display, noting that:
```
